.coveragerc

```diff
@@ -0,0 +1,7 @@
+[run]
+branch = True
+source = AccessControl
```

Comment:
Good catch, I copied the file from there and this line seem to have slipped through my fingers. But it seems that `AccessControl` here does not harm. 
Changed it anyway.